com.elsevier.MendeleyDesktop.json

```diff
@@ -0,0 +1,84 @@
+{
+    "app-id": "com.elsevier.MendeleyDesktop",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
```

Comment:
Does it use system Qt or bundles its own?

Reply:
No. Removed